nautobot/extras/templates/extras/job_approval_request.html

```diff
@@ -0,0 +1,152 @@
+{% extends 'base.html' %}
```

Comment:
Is it feasible/desirable for `job_approval_request.html` and `job.html` to share a common base template? Seems like a lot of duplication between these two.

Reply:
I feel like they’re just different enough where merging them might be tricky and somewhat clunky. I can certainly try it out, though!

Reply:
I tried to unify this, but didn’t find a nice way to do it—it comes out rather clunky.

I’d either defer/drop this or request a pairing session to have a look together if that’s possible :)

Reply:
Thanks for investigating. I'm fine with dropping this for now.